webui/react/src/types.ts

```diff
@@ -257,13 +264,15 @@ export interface Step {
   state: RunState;
 }
 
+// FIXME this vs TrialItem
```

Comment:
I think we discussed this being different right? They can share a common base possibly of:
```
interface Trial {}
interface TrialItem extends Trial {}
interface TrialDetail extends Trial {}
```

Reply:
I went with TrialBase to separate with the other items which don't have the Item suffix.